lib/airbrake.js

```diff
@@ -54,10 +55,11 @@ Airbrake.PACKAGE = (function() {
   return JSON.parse(json);
 })();
 
-Airbrake.createClient = function(key, env) {
+Airbrake.createClient = function(key, projectId, env) {
```

Comment:
What do you think about making `projectId` first parameter? IMO it looks more natural.